dev/devicelab/bin/tasks/flutter_gallery_instrumentation_test.dart

```diff
@@ -16,9 +16,15 @@ Future<Null> main() async {
     final Directory galleryDirectory =
       dir('${flutterDirectory.path}/examples/flutter_gallery');
     await inDirectory(galleryDirectory, () async {
+      final Device device = await devices.workingDevice;
+      await device.unlock();
```

Comment:
The devices are configured to never lock automatically. Some of our tests run for >20 minutes and rely on them being unlocked all the time. Instead, [the devicelab infra locks the screen](https://github.com/flutter/cocoon/blob/c5e229aa8d7ff890493a0b0fb163cfd2cb27e0ad/agent/lib/src/commands/ci.dart#L104) when the test is done.